types/rebass/index.d.ts

```diff
@@ -18,7 +18,10 @@ type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
 
 export interface BaseProps extends React.Props<any> {
     as?: React.ReactType;
-    css?: StyledComponents.CSSObject;
+    css?:
+        | StyledComponents.CSSObject
+        | StyledComponents.BaseThemedCssFunction<any>
```

Comment:
Though it is not mandatory, adding test case for `StyledComponents.BaseThemedCssFunction` would make this patch even better.

Reply:
I'll add a test case right now.

Reply:
Ah whoops I realized that `StyledComponents.BaseThemedCssFunction` isn't returned to the `css` prop but `StyledComponents.FlattenSimpleInterpolation` is.  Will fix

Reply:
Fixed